src/GT.UI/Program.cs

```diff
@@ -142,10 +142,10 @@
 app.UseAuthentication();
 app.UseAuthorization();
 
-app.MapControllerRoute(
-		name: "OnlyAction", // Route name
-		pattern: "/{action}", // URL with parameters
-		defaults: new { controller = "Home", action = "Index" }); // Parameter defaults
+//app.MapControllerRoute(
+//		name: "OnlyAction", // Route name
+//		pattern: "/{action}", // URL with parameters
+//		defaults: new { controller = "Home", action = "Index" }); // Parameter defaults
```

Comment:
Should add a TODO comment if these are going to be left commented out for an undisclosed amount of time.